File: SQL/sql.py
from __future__ import print_function

from pyspark.sql import SparkSession
from pyspark.sql import Row



if __name__ == "__main__":
    filein = sys.argv[0]
    fileout = sys.argv[1]

    spark = SparkSession \
        .builder \
        .appName("Python Spark SQL basic example") \
        .getOrCreate()



    df = spark.read.json(filein)
    result = df.select("Arrival_Time").mean
    result.saveAsTextFile(fileout)

File: SQL/sql.scala
import org.apache.spark.sql.SparkSession
import org.apache.spark.sql.Row

def main(args: Array[String]) {
  val filein = args(0)
  val fileout = args(1)

  val spark = SparkSession
    .builder()
    .appName("Spark SQL basic example")
    .getOrCreate()

  val df = spark.read.json(filein)
